RepoCleanup/Functions/CreateOrgWithTeams.cs

```diff
@@ -0,0 +1,152 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Text.Json;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using RepoCleanup.Models;
+using RepoCleanup.Services;
+
+namespace RepoCleanup.Functions
+{
+    public static class CreateOrgWithTeams
+    {
+
+        public static async Task Run()
+        {
+            StringBuilder logBuilder = new StringBuilder();
+            string logName = @$"CreateOrgWithTeams-Log.txt";
+            bool createdOrg = false;
+
+            Console.Clear();
+            Console.WriteLine("\r\n----------------------------------------------------------------");
+            Console.WriteLine("------------ Create new organisation with teams ----------------");
+            Console.WriteLine("----------------------------------------------------------------");
+
+            Organisation org = CollectOrgInfo();
+
+            Console.WriteLine("Creating org. Please wait.");
+            GiteaResponse response = await GiteaService.CreateOrg(org);
+
+            if (response.Success)
+            {
+                logBuilder.AppendLine($"{DateTime.Now} - Information - Org {org.Username} was successfully added to Gitea.");
+                createdOrg = true;
+            }
+            else
+            {
+                logBuilder.AppendLine($"{DateTime.Now} - Error - Org {org.Username} was not added to Gitea.");
+                logBuilder.AppendLine($"{DateTime.Now} - Error -  {response.StatusCode}:{JsonSerializer.Serialize(response.ResponseMessage, Globals.SerializerOptions)}.");
+            }
+
+            using (StreamWriter file = new StreamWriter(logName, true))
+            {
+                file.WriteLine(logBuilder.ToString());
+            }
+
+            logBuilder.Clear();
+
+            if (createdOrg)
+            {
+                Console.WriteLine($"Org {org.Username} successfully created.");
+            }
+            else
+            {
```

Comment:
Should the logic stop here? If no org is created there is no need to create the teams either..

Reply:
If org already is created it will fail and go on, but it will still create the default teams if they don't exists.